src/index.html

```diff
@@ -11,8 +11,78 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      rel="stylesheet"
+      href="./stars--0.css"
+    />
+    <link
+      rel="stylesheet"
+      href="./stars--1.css"
+    />
+    <link
+      rel="stylesheet"
+      href="./stars--2.css"
+    />
+    <link
+      rel="stylesheet"
+      href="./stars--3.css"
+    />
+    <link
+      rel="stylesheet"
+      href="./stars--4.css"
+    />
+    <link
+      rel="stylesheet"
+      href="./stars--5.css"
+    />
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars stars--0">
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+    </div>
```

Comment:
The `stars--0` block should have no active stars. Ensure that the CSS for this block reflects this requirement.